### DISCUSSION TITLE
Change option's background color when keypressed

### DISCUSSION THREAD

**Discussion by nsoubelet** [JedWatson/react-select#5338] [Q&A]
Hi there,
I am not able to identify how to change the default light blue background color when key is pressed over an option.

Is there any state I'm missing here?
`backgroundColor: state.isSelected ? "green" : state.isFocused ? 'green' : state.isActive ? "red" : "bg-gray-50",`
I thought it was the effect of `active`, but apparently it is not.

I also tried this with no luck:
```
'&:active': {
    ...styles,
    backgroundColor: "red",
    color: "white"
}
```
It's strange because `color` works in this case, but `backgroundColor` is not being picked up...
Any idea? Thanks

**Comment by manjushsh** [comment]
Can you share a sandbox link with issue replicated?
Something [like this?](https://codesandbox.io/s/customize-css-in-react-select-nkff64?file=/src/App.js)

**Reply by nsoubelet** [reply]
Yes, [here](https://codesandbox.io/s/customize-css-in-react-select-forked-vp01fh?file=/src/App.js).
I just forked yours and added a `style2` which is applied to 2nd `Select`, basically my idea is to change the BG color when option clicked, instead of using the primary25.

I solved the issue by using a custom template, which is not really good solution but it works.
```
theme={(theme) => ({
    ...theme,
    borderRadius: 0,
    colors: {
        ...theme.colors,
        primary25: "my-color1",
        primary50: ""my-color2",
        primary: ""my-color2",
    },
})}
```

**Reply by manjushsh** [reply]
yep
